DSCResources/MSFT_xADRecycleBin/Docs/readme.md

```diff
@@ -0,0 +1,15 @@
+The xADRecycleBin DSC resource will enable the Active Directory Recycle Bin feature for the target forest.
```

Comment:
Yes, please change README.md in the root. I'm apologies again, we are still converting documentation (https://github.com/PowerShell/DscResources/issues/2). Thank you for the early adoption!